app/controllers/templates/template.js

```diff
@@ -0,0 +1,69 @@
+import Controller from '@ember/controller';
+import { A } from '@ember/array';
+import { computed } from '@ember/object';
+import { sum, filterBy, mapBy } from '@ember/object/computed';
+
+export default Controller.extend({
+  achievementsModel: computed('game.achievements', function() {
+    let ret = A();
+    for (var u of this.game.achievements.values()) {
+      ret.pushObject(u)
+    }
+    return ret
+  }),
+
+  activeAchievements: filterBy('achievementsModel', 'isActive', true),
+  templatePointsArray: mapBy('activeAchievements', 'templatePoint'),
+  templatePoints: sum('templatePointsArray'),
+  remainingTemplatePoints: computed('templatePoints', 'model.{popTP,foodTP}', function() {
+    //TODO: Find a more dynamic way to compute this.
+    return this.templatePoints - (this.model.popTP+this.model.foodTP)
+  }),
+
+  rebirthPop: computed('model.popTP', function() {
+    return 1+this.model.popTP
+  }),
+
+  rebirthFood: computed('model.foodTP', function() {
+    return 10*this.model.foodTP
+  }),
+
+  actions: {
+    async lessPop(event) {
```

Comment:
This is quite boring. We will need a component for adding/removing, as this will happen in a few places in the game.

Reply:
Note created in kanban